exp/api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -206,6 +206,8 @@ type ManagedControlPlaneVirtualNetwork struct {
 	CIDRBlock string `json:"cidrBlock"`
 	// +optional
 	Subnet ManagedControlPlaneSubnet `json:"subnet,omitempty"`
+	// ResourceGroupName is the name of the Azure resource group for the VNet and Subnet.
```

Comment:
I assume we want this to be an optional field?

Reply:
will update accordingly.